interface/forms/eye_mag/view.php

```diff
@@ -221,14 +221,17 @@ function sel_diagnosis(target,term) {
     </script>
 
     <!-- Add Font stuff for the look and feel.  -->
+    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
+    <?php if ($_SESSION['language_direction'] == 'rtl') { ?>
+        <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-rtl-3-3-4/dist/css/bootstrap-rtl.min.css">
+    <?php } ?>
```

Comment:
Need to test the eye stuff in rtl (if looks funny, then will revert the rtl stuff in this and above eye forms)
TODO